### PR TITLE
adding JFrog Mission control

### PR DESCRIPTION
**Pull request by jainishshah17**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by wwsean08** [comment]
One suggestion for mission control which I opened with JFrog support and they pointed me here, was explicitly setting the namespace of the objects, see case 82862.

**Comment by eldada** [comment]
Team - can anyone take this? It's waiting for almost a month! @unguiculus, @mattfarina - can you help?

**Comment by unguiculus** [comment]
@jainishshah17 As suggested on Slack and on your other PRs, please compare with what you get from `helm create` using Helm 2.8+ and update accordingly.

**Comment by jainishshah17** [comment]
@unguiculus Updated. Can you please take a look now.

**Comment by jainishshah17** [comment]
@unguiculus Thank you for helping with other PRs. This is the last one.

File: incubator/mission-control/templates/hooks/create-user.yaml
Comment (unguiculus) [inline incubator/mission-control/templates/hooks/create-user.yaml]: app: {{ template "mission-control.name" . }}

File: incubator/mission-control/templates/hooks/create-user.yaml
Comment (unguiculus) [inline incubator/mission-control/templates/hooks/create-user.yaml]: Resource names should be based on the fullname template.

File: incubator/mission-control/templates/hooks/create-user.yaml
Comment (unguiculus) [inline incubator/mission-control/templates/hooks/create-user.yaml]: Please use the `chart` partial everywhere.

File: incubator/mission-control/templates/insight-executor-deployment.yaml
Comment (unguiculus) [inline incubator/mission-control/templates/insight-executor-deployment.yaml]: You should update deployments to `apps/v1beta2`

File: incubator/mission-control/templates/mission-control-certs.yaml
Comment (unguiculus) [inline incubator/mission-control/templates/mission-control-certs.yaml]: Shouldn't the user be able to bring their own key? Also, keys should probably go into a secret.

File: incubator/mission-control/charts/elasticsearch/templates/_helpers.tpl
Comment (unguiculus) [inline incubator/mission-control/charts/elasticsearch/templates/_helpers.tpl]: Please update to the latest `_helpers.tpl`.

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/mission-control/templates/mission-control-certs.yaml
Comment (jainishshah17) [reply incubator/mission-control/templates/mission-control-certs.yaml]: As of now, this is the only way to provide keys. I tried to convert this to a secret. We are going to have master key concept in this one as well which will eliminate need of passing keys like this.

File: incubator/mission-control/charts/elasticsearch/templates/_helpers.tpl
Comment (jainishshah17) [reply incubator/mission-control/charts/elasticsearch/templates/_helpers.tpl]: Done

File: incubator/mission-control/templates/hooks/create-user.yaml
Comment (jainishshah17) [reply incubator/mission-control/templates/hooks/create-user.yaml]: done

**Comment by unguiculus** [comment]
I'd suggest you move the chart to stable before we merge. As mentioned on Slack, I still don't like the fact that there are default keys in the chart. Can't you require the user to provide them? For CI you can add a separate values file to provide them.

**Comment by jainishshah17** [comment]
@unguiculus How about we keep this keys as default and we document the script to generate this keys?

**Comment by jainishshah17** [comment]
@unguiculus I have removed default keys and provided a way for the user to generate them. Please take a look.

File: incubator/mission-control/values.yaml
Comment (unguiculus) [inline incubator/mission-control/values.yaml]: To follow a common practice, please rename this to `existingCertsSecret`.

**Review by unguiculus — CHANGES_REQUESTED** [review]
I'd suggest you move the chart to stable before we merge it.

File: incubator/mission-control/values.yaml
Comment (jainishshah17) [reply incubator/mission-control/values.yaml]: Done

**Comment by jainishshah17** [comment]
@unguiculus Waiting for your final response.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**
